test/integration/TestBase.sol

```diff
@@ -167,4 +199,39 @@ contract TestBase is Test {
         vm.prank(_migrationAdmin);
         _wrappedMToken.migrate(_wrappedMTokenMigratorV1);
     }
+
+    /* ============ utils ============ */
+
+    function _makeKey(string memory name_) internal returns (uint256 key_) {
+        (, key_) = makeAddrAndKey(name_);
```

Comment:
Could just use `(address _alice, uint256 _aliceKey) = makeAddrAndKey("alice");` instead of introducing this helper.

Reply:
Yes, but then how will I do this nicely in the global variables?

```
address internal _alice = makeAddr("alice");
uint256 internal _aliceKey = _makeKey("alice");
```

Sadly, solidity does not support

```
(address internal _alice, uint256 internal _aliceKey) = makeAddrAndKey("alice");
```